storage/object/api-cli/bucket-policy.mdx

```diff
@@ -282,23 +244,39 @@ Bucket policies use a JSON-based access policy language and are composed of stri
     ]
 }
 ```
+
+### NotPrincipal
+
+**Description**
+: Everybody **except** the [user](/identity-and-access-management/iam/concepts#user) or [application](/identity-and-access-management/iam/concepts#application) targeted by the bucket policy.
+
+**Required**
+: Yes
+
+**Parent**
+: `Statement`
+
+**Supported values**
+: `user_id` / `application_id` / `*`
+
+**Sample**
+
 ```json
 {
-    "Version": "2023-04-17",
+    "Version": "2023-06-07-alpha",
     "Id":"MyBucketPolicy",
     "Statement": [
         {
-            "Sid": "Grant List and GET to user and application",
-            "Effect":"Allow",
-            "Principal":{
+            "Sid": "Deny access to all except user and application",
+            "Effect":"Deny",
+            "NotPrincipal":{
                 "SCW":[
                     "user_id:<USER_ID>",
```

Comment:
This has to be discussed since we said yesterday with design it was impossible - yet it is supported